pyccel/codegen/printing/ccode.py

```diff
@@ -691,14 +703,15 @@ def _print_ModuleHeader(self, expr):
                 '#define {name}_H\n\n'
                 '{imports}\n'
                 '{variables}\n'
-                #'{classes}\n'
+                '{classes}\n'
                 '{funcs}\n'
                 #'{interfaces}\n'
                 '#endif // {name}_H\n').format(
                         name    = name.upper(),
                         imports = imports,
                         variables = global_variables,
-                        funcs   = funcs)
+                        funcs   = funcs,
+                        classes = classes)
```

Comment:
Codacy won't like this. Please use fstrings

Reply:
Fixed